### PR TITLE
chore(readme): Remove extra developer configuration steps

### PR DESCRIPTION
**Pull request by meriley**
README Changes:
1. Prettier on README.md
2. Removed extra dev steps

Script Changes:
1. Use node_modules type script.

The only npm global required is 

```
PS > npm list -g --depth=0
C:\Users\meril\AppData\Roaming\npm
`-- windows-build-tools@5.2.2
```

### CODE REVIEW

**Comment by meriley** [comment]
Readability in mark down I think is improved, and but really because Prettier is installed on my VScode :p

File: README.md [hunk 1]
```diff
@@ -135,35 +138,28 @@ Your editor of choice for a node project - like [vscode](https://code.visualstud
...
 
-A TypeScript compiler:
-```
-npm install -g typescript
 ```
-
-And the Angular CLI:
-```
-npm install -g @angular/cli
+npm install --global --production windows-build-tools
```
Comment (meriley) [inline README.md:143]: This is the core change in the read me that is not just formatting.

Ive removed installation of other global packages,

**Comment by ijemafe** [comment]
For additional context, I added the instructions to install typescript and angular CLI globally in my fork. More experienced devs have helped me find a better way to use these packages locally instead of installing them globally.

Testing now.

**Comment by ijemafe** [comment]
Confirmed this works perfectly, thanks!

**Pull request merged**